designer/server/src/models/account/auth.js

```diff
@@ -1,5 +1,17 @@
+import config from '~/src/config.js'
+
 export function signedOutViewModel() {
   return {
     pageTitle: 'You have signed out'
   }
 }
+
+/**
+ *  @param {boolean} userFailedAuthorisation - indicates the user has failed authorisation, but authentication was successful
+ */
+export function signInViewModel(userFailedAuthorisation) {
+  return {
+    pageTitle: `Sign in to ${config.serviceName}`,
```

Comment:
Thanks for moving more bits into view models 🙌

Not a blocker, but might need to customise this `pageTitle` when auth scopes prevent sign in

Reply:
Done

Reply:
Done

Reply:
Not done, never mind.

Reply:
Done